extension/EdFi.Ods.Extensions.LMSX/LMS-Harmonizer/1090-lms-assignment-submissions.sql

```diff
@@ -20,9 +20,16 @@ BEGIN
 		lmsSubmission.LastModifiedDate,
 		lmsSubmission.DeletedAt
 	INTO #ALL_SUBMISSIONS
-	FROM LMS.AssignmentSubmission lmsSubmission
-	INNER JOIN LMSX.Assignment lmsxAssignment
-		ON lmsSubmission.AssignmentIdentifier = lmsxAssignment.AssignmentIdentifier
```

Comment:
We definitely made a mistake in our column naming - this _looks_ correct but is in fact wrong. the value on the left is an integer, while the value on the right is a string - it is the Assignment's Source System Identifier. In order to write the correct query, I had to add a join from `lms.AssignmentSubmission` to `lms.Assignment`, and then the join to `lmsx.Assignment` (the extension table) goes through `lms.Assignment` instead of trying to directly join `lms.AssignmentSubmission` to `lmsx.Assignment`.